py/src/braintrust/logger.py

```diff
@@ -60,6 +60,8 @@
 Metadata = Dict[str, Any]
 DATA_API_VERSION = 2
 
+BRAINTRUST_APP_URL = os.environ.get("BRAINTRUST_APP_URL", "https://www.braintrust.dev")
```

Comment:
should this be `GLOBAL_BRAINTRUST_APP_URL`? if so, don't forget to adjust the import in `py/src/braintrust/cli/install/api.py`

Reply:
Whoops. No, I think what we want to do here is to compute the env var in global scope, and use that in both places. So should use `BRAINTRUST_APP_URL` everywhere.